plasmapy/physics/transport/collisions.py

```diff
@@ -689,6 +700,235 @@ def collision_frequency(T,
     return freq.to(u.Hz)
 
 
+@utils.check_quantity({
+    'T_e': {'units': u.K, 'can_be_negative': False},
+    'n_e': {'units': u.m ** -3, 'can_be_negative': False}
+    })
+def collision_rate_electron_ion(T_e,
+                                n_e,
+                                ion_particle,
+                                coulomb_log=None,
+                                V=None,
+                                coulomb_log_method="classical"):
+    r"""
+    Momentum relaxation electron-ion collision rate
+
+    From [3]_, equations (2.17) and (2.120)
+
+    Considering a Maxwellian distribution of "test" electrons colliding with
+    a Maxwellian distribution of "field" ions.
+
+    This result is an electron momentum relaxation rate, and is used in many
+    classical transport expressions. It is equivalent to:
+    * 1/tau_e from ref [1]_ eqn (1) pp. #,
+    * 1/tau_e from ref [2]_ eqn (1) pp. #,
+    * nu_e\i_S from ref [2]_ eqn (1) pp. #,
+
+    Parameters
+    ----------
+    T_e : ~astropy.units.Quantity
+        The electron temperature of the Maxwellian test electrons
+
+    n_e : ~astropy.units.Quantity
+        The number density of the Maxwellian test electrons
+
+    ion_particle: str
+        String signifying a particle type of the field ions, including charge
+        state information.
+
+    V : ~astropy.units.Quantity, optional
+        The relative velocity between particles.  If not provided,
+        thermal velocity is assumed: :math:`\mu V^2 \sim 2 k_B T`
+        where `mu` is the reduced mass.
+
+    coulomb_log : float or dimensionless ~astropy.units.Quantity, optional
+        Option to specify a Coulomb logarithm of the electrons on the ions.
+        If not specified, the Coulomb log will is calculated using the
+        `~plasmapy.physics.transport.Coulomb_logarithm` function.
+
+    coulomb_log_method : string, optional
+        Method used for Coulomb logarithm calculation (see that function
+        for more documentation). Choose from "classical" or "GMS-1" to "GMS-6".
+
+    References
+    ----------
+    .. [1] Braginskii
+
+    .. [2] Formulary
```

Comment:
If you have the time, it would be good to add in the full references here including a link to the Formulary.  Alternatively, if this comes up in a bunch of places, we could create an issue for it.